src/components/co-sign/CoSign.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { ReactNode } from 'react'
+
+import { StyleProp, StyleSheet, View, ViewStyle } from 'react-native'
+
+import IconCoSign from 'app/assets/images/iconCoSign.svg'
+import { useThemeColors } from 'app/utils/theme'
+
+import { Size } from './types'
+
+type CoSignProps = {
+  size: Size
+  children: ReactNode
+  style?: StyleProp<ViewStyle>
+}
+
+const CoSign = ({ size, children, style }: CoSignProps) => {
+  const { primary, staticWhite } = useThemeColors()
+  const styles = StyleSheet.create({
```

Comment:
I might be wrong, but it's not kosher to do stylesheet.create inside render function? my assumption is that you can pass arguments to the Stylesheet.create in the same way we pass the theme in?

Reply:
Oh yeah you're right, I think it would make sense to apply the styles inline in this case. Similar to how it's done for the icon

Reply:
I thought about this more and I'm pretty sure it is kosher to do stylesheet.create in the render function. We're already doing that (memoized) every time we use `useThemedStyles`

The main benefit to using stylesheet.create  is to get type checking of styles before you apply them to a component, I don't think there's anything expensive happening in there

I'm still gonna update this though because it will be cleaner without it